pom.xml

```diff
@@ -176,35 +196,101 @@
-                </condition>
-              </target>
-            </configuration>
-            <goals>
-              <goal>run</goal>
-            </goals>
-          </execution>
-        </executions>
-      </plugin>
     </plugins>
   </build>
 
   <profiles>
+    <!-- Build distribution statically linked to BoringSSL -->
+    <profile>
+      <id>boringssl</id>
+
+      <properties>
+        <artifactIdToUse>netty-tcnative-boringssl</artifactIdToUse>
+        <boringSslHome>${project.build.directory}/boringssl</boringSslHome>
+        <checkoutDirectory>${boringSslHome}</checkoutDirectory>
+        <boringSslBuildDir>${boringSslHome}/build</boringSslBuildDir>
+        <boringSslVersion>7b8b9c17db93ea5287575b437c77fb36eeb81b31</boringSslVersion>
+      </properties>
+
+      <build>
+        <plugins>
+          <!-- Download the BoringSSL source -->
+          <plugin>
+            <artifactId>maven-scm-plugin</artifactId>
+            <version>1.9.4</version>
+            <executions>
+              <execution>
+                <id>get-boringssl</id>
+                <phase>generate-sources</phase>
+                <goals>
+                  <goal>checkout</goal>
+                </goals>
+                <configuration>
+                  <connectionType>developerConnection</connectionType>
+                  <developerConnectionUrl>scm:git:https://boringssl.googlesource.com/boringssl</developerConnectionUrl>
+                  <scmVersion>${boringSslVersion}</scmVersion>
+                  <scmVersionType>revision</scmVersionType>
+                </configuration>
+              </execution>
+            </executions>
+          </plugin>
+
+          <!-- Build BoringSSL -->
+          <plugin>
+            <artifactId>maven-antrun-plugin</artifactId>
+            <executions>
+              <execution>
+                <id>make-boring-ssl</id>
+                <phase>generate-sources</phase>
+                <goals>
+                  <goal>run</goal>
+                </goals>
+                <configuration>
+                  <target name="makeBoringSsl">
+                    <mkdir dir="${boringSslBuildDir}" />
+                    <exec executable="cmake" failonerror="true" dir="${boringSslBuildDir}" resolveexecutable="true">
+                      <arg value=".." />
+                      <arg value="-DCMAKE_POSITION_INDEPENDENT_CODE=TRUE" />
+                    </exec>
+                    <exec executable="make" failonerror="true" dir="${boringSslBuildDir}" resolveexecutable="true"/>
+                  </target>
+                </configuration>
+              </execution>
+            </executions>
+          </plugin>
+
+          <!-- Configure the distribution statically linked against BoringSSL -->
+          <plugin>
+            <groupId>org.fusesource.hawtjni</groupId>
+            <artifactId>maven-hawtjni-plugin</artifactId>
+            <version>1.11</version>
```

Comment:
use a property for the version and use the property in both places where you specified the plugin ?